### DISCUSSION TITLE
Do alert rules support variables?

### DISCUSSION THREAD

**Discussion by FengZh61** [ccfos/nightingale#1245] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/98309003/199151867-05dbbd3f-957b-4c90-9766-47f7c3ded539.png)
类似于prometheus 的 alert rules。

**Comment by UlricQin** [comment, marked as answer]
`{{$labels.ident}}` 最新版本，告警规则标题和备注都支持这么写
